meta/main.yml

```diff
@@ -1,8 +1,9 @@
 ---
 
 dependencies:
-  - name: adfinis.icinga2_agent
-
+  - role: adfinis.icinga2_agent
+collections:
+  - community.mysql
```

Comment:
Based on [this SO discussion](https://stackoverflow.com/questions/69508170/ansible-role-dependency-handling-for-collections) and [this GH issue](https://github.com/ansible/ansible/issues/76030), roles cannot state collections in their `main/meta.yml` file.

I'm not sure this is going to be accepted by whatever processes this.

Reply:
(as discussed off-channel, let's go with this anyway)